docs/source/conf.py

```diff
@@ -17,21 +17,19 @@
 
 # -- Project information -----------------------------------------------------
 
-project = "NYUAD MRI Lab Documentation"
-copyright = "2022, Haidee Paterson, Osama Abdullah"
-author = "Haidee Paterson, Osama Abdullah"
+project = 'MRI Spectroscopy Documentation'
```

Comment:
These details should not be changed, your content is added to the existing MRI website and not as a separate website if I understood correctly, please set them back